zhaquirks/philips/zlldimmablelight.py

```diff
@@ -22,7 +22,12 @@
     PROFILE_ID,
     MODELS_INFO,
 )
-from zhaquirks.philips import PHILIPS, PhilipsOnOffCluster
+
+from zhaquirks.philips import (
+    PHILIPS,
+    PhilipsLevelControlCluster,
+    PhilipsOnOffCluster
+)
```

Comment:
``from zhaquirks.philips import PHILIPS, PhilipsLevelControlCluster, PhilipsOnOffCluster``

Reply:
I put them on separate lines the same as zllextendedvolorlight.py
https://github.com/zigpy/zha-device-handlers/blob/6682592d6b3935e495503ab20177312b8dd6b55a/zhaquirks/philips/zllextendedcolorlight.py#L26-L31

Reply:
PhilipsColorCluster not being needed here makes it only 3 (not 4) imports. Hence the shorter form should be used. That’s also why the lint fails.

Reply:
Lint also tells you what it wants to reformat, e.g. https://travis-ci.org/github/zigpy/zha-device-handlers/jobs/740724957#L240

But just use black and  install pre-commit
```
pip3 install -U pre-commit
pre-commit install
```